config/tasks/task_steps_config-recipe_m2_apply_tourniquet.yaml

```diff
@@ -0,0 +1,112 @@
+# Schema version.
+version: "1.0"
+
+# Reference to the activity classification labels configuration that we will
+# reference into.
+activity_labels: "./config/activity_labels/recipe_m2_apply_tourniquet.yaml"
```

Comment:
This config doesn't exist yet in the repo. Is this file intended to be added to this branch?